### PR TITLE
CC-25 Add styling for main content - Dashboard component

### PR DESCRIPTION
**Pull request by skrzepij**
* Add Font Awesome; 
* Add styles for main content section; 
* Add background image; 
* Add translation component for PL language; 
* Add couple of generic styling files like mixins, buttons, forms; 
* Refactor html structure of elements;

### CODE REVIEW

**Comment by skrzepij** [comment]
[(5) CC-25 Dodać style dla main contentu / Dashboard](https://trello.com/c/Duzt9Xr5/60-5-cc-25-doda%C4%87-style-dla-main-contentu-dashboard)

**Comment by skrzepij** [comment]
[CC-26 Style - użyć grida](https://trello.com/c/VPMEquYa/32-cc-26-style-u%C5%BCy%C4%87-grida)

File: src/styles/vendors/_antd.scss [hunk 1]
```diff
@@ -39,3 +41,26 @@
...
+// Grid
+.ant-layout-content {
+  padding: 0 $gap-big;
+}
+
+// Forms
+.ant-select {
+  $input-height: 50px;
+  .ant-select-selection {
+    &--single {
```
Comment (MateuszWasik) [inline src/styles/vendors/_antd.scss:54]: what `--single` does ?

**Review by MateuszWasik — APPROVED** [review]

File: src/components/Dashboard.js [hunk 2]
```diff
@@ -55,43 +55,61 @@ export const Dashboard = () => {
   }
 
   return (
-    <div className="converter-container">
-      <div className="selector-wrapper">
+    <div className="converter cc-container">
+      <div className="converter__info">
+        <div className="converter__info__content">
+          <p>
+            Wyliczenia wykonywane są na podstawie najnowszych danych, udostępnionych przez <strong>Narodowy Bank Polski</strong>.
```
[the remark this answers is not in the document]
Comment (skrzepij) [reply src/components/Dashboard.js:62]: ja ja. Can do.

File: src/styles/vendors/_antd.scss (hunk 1 above)
Comment (skrzepij) [reply src/styles/vendors/_antd.scss:54]: it's additional class added by default from `ant design` for each ant select which is single type. 
`.ant-select-selection--single`

File: src/components/Dashboard.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (skrzepij) [reply src/components/Dashboard.js:62]: discussed f2f. will be changed during the translation task.

**Pull request merged**